src/compiler/crystal/macros.cr

```diff
@@ -100,19 +100,19 @@ module Crystal::Macros
 
   # Skips the rest of the file from which it is executed.
   # Typical usage is to skip files that have platform specific code,
-  # without having to surround the most relevant code in `{%...%}` macro blocks.
+  # without having to surround the most relevant code in `{% if flag %} ... {% end %}` macro blocks.
   #
   # Example:
   #
   # ```
   # # sth_for_osx.cr
-  # {% skip unless flag?(:darwin) %}
+  # {% skip_file unless flag?(:darwin) %}
```

Comment:
AFAIK it *requires* `skip_file()` with the empty brackets.

Reply:
True. So this example was already incorrect with `skip` ...